OneCommunityGlobal/HighestGoodNetworkApp
# Jay - PERMISSION: Add Time Entry (Others)

# Description
This PR enables the user with the Add Time Entry (Others) permission to be able to add a time entry form on any user via their dashboard.

## Related PRS (if any):
This frontend PR is related to the [#727](https://github.com/OneCommunityGlobal/HGNRest/pull/727) backend PR.

…

## Main changes explained:
- Added conditional logic so that the user with the permission can access and submit the time entry from using others' dashboard.

## How to test (please watch the video):
1. check into current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log in with an owner account
5. go to Other Links -> Permissions Management -> Manage User Permissions -> Choose a USER without **Add Time Entry (Others)** permission under Timelog Management -> click on Add  -> Scroll down then submit
6. verify that the user who was given the permission is now able to see time entry button on another user's dashboard and submit it.

## Screenshots or videos of changes:




https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/118868270/0c3bbd79-4b40-421b-9e39-ea7502bb9f06






## Note:
**Unmute the video**
